frontend/src/components/ViewActivities/editActivity.js

```diff
@@ -96,17 +100,33 @@ class EditActivity extends React.Component {
    */
   countriesDropdown(ref, tzref) {
     return (
-      <Form.Control as="select" ref={ref} onChange={tzref}>
-        <option key="-1">No Change</option>
+      <Form.Control as='select' ref={ref} onChange={tzref}>
+        <option key='-1'>No Change</option>
```

Comment:
What is the purpose of the -1 key? I couldn't find anything in the Form ReactBootstrap documentation.

Reply:
The -1 keeps that "no change" item at the top of the list. It's replaced with something more intuitive when the form autofills with previously known data in #86.